Python/Temperature_Conversion/README.md

```diff
@@ -0,0 +1,30 @@
+# Temperature Conversion
+
+## Description
+A python script to convert temperature from and to Celsius, Kelvin or Fahrenheit scales
+
+## Setup Instructions
+
+1. Run the python script 
+
+2. Enter the choices and temperature to be converted
+
+
```

Comment:
Remove these excess whitespaces, 1 is sufficient

Reply:
Sure!